scripts/soroban-settings/SorobanSettingsUpgrade.py

```diff
@@ -0,0 +1,258 @@
+from stellar_sdk import xdr as stellar_xdr
+from stellar_sdk import Network, Keypair, TransactionBuilder, StrKey, utils
+from stellar_sdk.soroban import SorobanServer
+from stellar_sdk.soroban.types import Address, Int128, Bytes
+from stellar_sdk.soroban.soroban_rpc import GetTransactionStatus
+from stellar_sdk.xdr import TransactionMeta, LedgerKey, ConfigUpgradeSet, ConfigSettingContractExecutionLanesV0, ConfigUpgradeSetKey, ConfigSettingEntry, StateExpirationSettings, Uint32, Uint64, Int64, Hash, LedgerKeyConfigSetting, ConfigSettingID
+import stellar_sdk
+from enum import IntEnum
+import urllib.parse
+import argparse
+import time
+import sys
+
+# The soroban branch of py-stellar-base hasn't been merged into main and released yet, so we have to install it locally.
+sys.path.append("/Users/dev/py-stellar-base")
```

Comment:
you can install from the branch via `pip`, if you want to add that to the instructions

```bash
pip install git+https://github.com/StellarCN/py-stellar-base.git@soroban
```

Reply:
Wow I'm a python noob. This is good to know. Thanks!

I removed this line [here](https://github.com/sisuresh/stellar-core/tree/upgrade-script) and added instructions yesterday for installing from source but the PR isn't updating for some reason.

Reply:
Heh this is thanks to @overcat, from here: https://github.com/stellar/sorobanathon/discussions/20. I do see it's gone from the branch, though :+1: